automated/linux/ota-rollback/download-update.sh

```diff
@@ -86,12 +87,13 @@ fi
 cp aklite-callback.sh /var/sota/
 chmod 755 /var/sota/aklite-callback.sh
 
-mkdir -p /etc/sota/conf.d
+mkdir -p "${SOTA_CONFDIR}"
+cp z-99-aklite-callback.toml "${SOTA_CONFDIR}"
+cp z-99-aklite-disable-reboot.toml "${SOTA_CONFDIR}"
 if [ "${PACMAN_TYPE}" = "ostree" ]; then
-    cp z-99-aklite-callback-ostree.toml /etc/sota/conf.d/
-else
-    cp z-99-aklite-callback.toml /etc/sota/conf.d/
+    cp z-99-ostree.toml "${SOTA_CONFDIR}"
 fi
+
```

Comment:
is this needed?
if yes, can add a newline at automated/linux/ota-update/download-update.sh row 99 also?